### PR TITLE
Adding Angular Material Design

### PR DESCRIPTION
**Pull request by rqton**
Done : Header & Lefter
Todo : Game Quizz

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-		"angular-resource"  : "~1.3.8",
-		"bootstrap"	        : "~3.3.1"
-	}
+  "name": "app-monitoring",
+  "dependencies": {
+    "jquery": "~1.11.2",
+    "angular": "~1.3.8",
+    "angular-route": "~1.3.8",
+    "angular-resource": "~1.3.8",
+    "bootstrap": "~3.3.1",
```
Comment (MythX) [inline bower.json]: You can remove this dependency because we don't use Bootstrap :)

**Comment by MythX** [comment]
It's ok for me. I don't have problem with code and the result is great.
I think it's better if you remove Bootstrap dependency in yout pull request.

File: bower.json [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {
-	"name": "app-monitoring",
-	"dependencies": {
-		"jquery"	        : "~1.11.2",
-		"angular"	        : "~1.3.8",
-		"angular-route"	    : "~1.3.8",
-		"angular-resource"  : "~1.3.8",
-		"bootstrap"	        : "~3.3.1"
-	}
+  "name": "app-monitoring",
```
Comment (xaviermichel) [inline bower.json]: What is "app-monitoring" ? ^^ (thanks AMO)

**Comment by xaviermichel** [comment]
:+1:

**Pull request merged**